crates/network-scheduler/src/prometheus_metrics.rs

```diff
@@ -0,0 +1,66 @@
+use std::collections::HashMap;
+
+use prometheus_client::metrics::gauge::Gauge;
+use prometheus_client::metrics::histogram::{linear_buckets, Histogram};
+use prometheus_client::registry::Registry;
+
+use crate::scheduling_unit::UnitId;
+
+lazy_static::lazy_static! {
+    static ref WORKERS_PER_UNIT: Histogram = Histogram::new(linear_buckets(0.0, 1.0, 100));
+    static ref TOTAL_UNITS: Gauge = Default::default();
```

Comment:
Maybe also add `TOTAL_DATA_SIZE`?

Reply:
I wanted to, but I thought that it should be recalculated each time we read a chunk from the storage, not here. So I'll create a separate PR for that.